File: unical-support-2/src/main/java/unical_support/unicalsupport2/data/dto/email/ClassifyDto.java
package unical_support.unicalsupport2.data.dto.email;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.util.List;

@Data
public class ClassifyDto {
    String id;
    private String explanation;
    @JsonProperty("isClassified") private boolean isClassified;
    private List<SingleClassificationDto> singleClassifications;
}

File: unical-support-interface/src/app/model/classify-dto.ts
import {SingleClassificationDto} from './single-classification-dto';

export class ClassifyDto {
  id: string;
  explanation: string;
  classified: boolean;
  singleClassifications: SingleClassificationDto[]

  constructor(data: any) {
    this.id = data.id;
    this.explanation = data.explanation;
    this.classified = data.classified;
    this.singleClassifications = data.singleClassifications;
  }
}
